src/main/java/uk/gov/hmcts/reform/sscs/messaging/HmcMessage.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.sscs.messaging;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class HmcMessage {
+    //TODO json object may need to change at a later stage when we get HMC data
+    private String hmctsServiceID;
+    private String caseRef;
+    private String hearingID;
+    private String hearingUpdate;
```

Comment:
Will `hearingUpdate` field will be changed from string to an object at a later date? (as per confluence page: https://tools.hmcts.net/confluence/display/HMAN/Message+Published+to+Services+by+HMC )

Reply:
![image](https://user-images.githubusercontent.com/15264837/160166238-174b857c-e82c-44a6-8ff4-8c92e8cffe2b.png)

Reply:
Im assuming these are strings for now.